client/src/remote/index.js

```diff
@@ -22,14 +22,10 @@ const {
   flags,
   platform,
   plugins: appPlugins,
-  ipcRenderer
+  api
```

Comment:
What is _api_  here? Can we give it a more sensible name? Alternative: Construct the Backend directly in appPreload. Not sure what's the benefit to construct it here?

Reply:
Benefits: 
* We don't expose the `ipcRenderer` methods directly.
* We can filter events for which `send` can be used.

Reply:
So yeah, I could rename it to `ipcRenderer` again. However, how does the client benefit from knowing about `ipcRenderer`?

Reply:
Does the client benefit from knowing something that is called API? 

A solution could be to return the instantiated backend from the pre-load function. This way we do not need to pass around these names at all. Let us try to avoid generics like "api" and "descriptor" in our code. We got a lot of these in there already :wink:

Reply:
Let's try that out.

Reply:
Implemented via https://github.com/camunda/camunda-modeler/pull/2220/commits/0a842176404bd81d2fe3353933111dedc2b3bc15